packages/cssnano-preset-advanced/test/browserslist/.browserslistrc

```diff
@@ -0,0 +1,12 @@
+defaults
+
+[legacy]
+IE <=11
+Chrome <=51
```

Comment:
Yeah I did, it was to find a nice support gap to test [postcss-colormin](https://www.npmjs.com/package/postcss-colormin) and [postcss-reduce-initial](https://www.npmjs.com/package/postcss-reduce-initial) since it was hard to find ranges that would clearly flip between `[modern]` and `[legacy]` environments etc

E.g. Chrome 47 to 61 via [**Browserslist** `supports css-placeholder-shown and not supports css-rrggbbaa`](https://browsersl.ist/#q=supports+css-placeholder-shown+and+not+supports+css-rrggbbaa)

I can't remember the exact query now, sorry